src/target/llvm/codegen_llvm.cc

```diff
@@ -927,6 +927,12 @@ llvm::Value* CodeGenLLVM::CreateIntrinsic(const CallNode* op) {
     value->addIncoming(then_value, then_value_block);
     value->addIncoming(else_value, else_value_block);
     return value;
+  } else if (op->op.same_as(builtin::ret())) {
+    ICHECK_EQ(Downcast<IntImm>(op->args[0])->value, 0);
```

Comment:
I believe we can also safely support other return values as int. 

We need to ICHECK op->args is an IntImm of type i32, return that constant value. Add an error message that we only support returning int value for now.